allensdk/test/brain_observatory/ecephys/test_copy_utility.py

```diff
@@ -99,14 +117,72 @@ def test_compare_files(tmpdir_factory, different, raise_if_comparison_fails):
 
         if raise_if_comparison_fails:
             with pytest.raises(ValueError):
-                cu.compare_files(spath, dpath, hasher_cls, raise_if_comparison_fails)
+                cu.compare_files(
+                    spath, dpath, hasher_cls, raise_if_comparison_fails)
         else:
             with pytest.warns(UserWarning):
-                cu.compare_files(spath, dpath, hasher_cls, raise_if_comparison_fails)
+                cu.compare_files(
+                    spath, dpath, hasher_cls, raise_if_comparison_fails)
 
     else:
 
         with open(dpath, 'w') as f:
             f.write('baz')
 
         cu.compare_files(spath, dpath, hasher_cls, raise_if_comparison_fails)
+
+
+def test_SessionUploadSchema(tmpdir):
+    src_file = Path(tmpdir) / 'src.csv'
+    src_file.touch()
+
+    dst_file = Path(tmpdir) / 'dst.csv'
+
+    output_json = Path(tmpdir) / 'output.json'
+
+    test_data = {
+        'files': [{
+            'source': str(src_file),
+            'destination': str(dst_file),
+            'key': ''
+        }],
+        'output_json': str(output_json)
+    }
+
+    parser = argschema.ArgSchemaParser(
+        test_data,
+        schema_type=SessionUploadInputSchema,
+        output_schema_type=SessionUploadOutputSchema,
+        args=[]
+    )
+
+    parser.output({'data': 0})
```

Comment:
your tests should fail, because `{'data': 0}` won't meet the validation criteria of the output schema.

Reply:
Uh oh. They don't fail locally...

Reply:
they should